modules/microshift-disabling-lvms-csi-driver.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/microshift-storage-plugin-overview.adoc
+// * microshift_configuring/microshift-disable-lvms-csi-provider-csi-snapshot.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-disabling-lvms-csi-driver_{context}"]
+= Disabling deployments that run the CSI driver implementations
+
+Use the following procedure to disable installation of the CSI implementation pods.
+
+[IMPORTANT]
+====
+This procedure is for users who are defining the configuration file before installing and running {microshift-short}. If {microshift-short} is already started then CSI driver implementation will be running. Users must manually remove it by following the uninstallation instructions.
+====
```

Comment:
Should this link to the uninstall instructions?

Reply:
done